packages/messaging/index.js

```diff
@@ -212,6 +213,9 @@ function getTransport (config, messagingContext) {
     if (config instanceof TestTransportConfig) {
         return new TestTransport(config, messagingContext)
     }
+    if (config instanceof SendMessageMessagingConfig) {
+        return new SendMessageMessagingTransport(config, messagingContext)
+    }
```

Comment:
As mentioned in our call, we can achieve this without touching the messaging framework :)